wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/core-customizer-css-preview.min.js

```diff
@@ -0,0 +1 @@
+!function(){var e,s;e=wp.customize,s=jQuery,e.settingPreviewHandlers?e.settingPreviewHandlers.custom_css=function(){}:parent.console.warn("Missing core patch that adds support for settingPreviewHandlers"),e.selectiveRefresh.partialConstructor.custom_css=e.selectiveRefresh.Partial.extend({refresh:function(){var t,r,c=e("jetpack_custom_css[preprocessor]").get();return jpCustomizerCssPreview.preprocessors.hasOwnProperty(c)?e.selectiveRefresh.Partial.prototype.refresh.call(this):(t=new s.Deferred,r=e("custom_css["+e.settings.theme.stylesheet+"]"),_.each(this.placements(),(function(e){e.container.text(r.get())})),t.resolve(),t.promise())}})}();
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).